makeflow/src/makeflow.c

```diff
@@ -874,7 +880,13 @@ static void makeflow_run( struct dag *d )
 	struct dag_node *n;
 	batch_job_id_t jobid;
 	struct batch_job_info info;
+        timestamp_t last_time = timestamp_get();
+        int first_report = 1;
 
+        //reporting to catalog
+        if(in_n_mode){
```

Comment:
I had uncertainty, because project is set some place other than just in the -N flag, so I wanted to have a separate, known variable that will only be true if the -N flag is passed